flocker/acceptance/scripts/lsblkhttp.py

```diff
@@ -0,0 +1,36 @@
+"""
+HTTP server that returns the size of the mounted volume.
+"""
+
+from subprocess import check_output
+
+from BaseHTTPServer import BaseHTTPRequestHandler, HTTPServer
+
+
+class Handler(BaseHTTPRequestHandler):
+    def do_GET(s):
+        s.send_response(200)
+        s.end_headers()
+        if "is_http" in s.path:
+            s.wfile.write(b"hi")
+        else:
+            try:
+                mounts = {}
+                for line in check_output(['mount', '-l']).split('\n'):
```

Comment:
:question: I was curious if `mount` had a stable output format, so I looked at mount(8), which pointed me to `findmnt`, which looks like it can get all the information we need in one call (presuming it is available).